server/src/app.module.ts

```diff
@@ -5,26 +5,31 @@ import { AppService } from './app.service';
 import { ConfigModule } from '@nestjs/config';
 import { CoreModule } from './core/core.module';
 import { ApolloDriver, ApolloDriverConfig } from '@nestjs/apollo';
+import { ApolloServerPluginLandingPageLocalDefault } from '@apollo/server/plugin/landingPage/default';
 import { GraphQLError } from 'graphql';
 import { PrismaModule } from './database/prisma.module';
 import { HealthModule } from './health/health.module';
+import { AbilityModule } from './ability/ability.module';
 
 @Module({
   imports: [
     ConfigModule.forRoot({
       isGlobal: true,
     }),
     GraphQLModule.forRoot<ApolloDriverConfig>({
+      playground: false,
       context: ({ req }) => ({ req }),
       driver: ApolloDriver,
       autoSchemaFile: true,
+      plugins: [ApolloServerPluginLandingPageLocalDefault()],
```

Comment:
@charlesBochet I use it because the interface is really better than the default `playground`, le me know if you want me to revert it

Reply:
<img width="2560" alt="Screenshot 2023-06-20 at 2 30 04 PM" src="https://github.com/twentyhq/twenty/assets/3551795/9117b515-01a7-429c-9d43-d64bee0f92e0">

Reply:
It's really better than the default playground I would advocate for it too.

Reply:
lgtm!